packages/dlm/dlm.0.3.1/opam

```diff
@@ -26,10 +27,9 @@ depexts: [
   ["dlm-devel"] {os-distribution = "centos"}
   ["dlm-devel"] {os-distribution = "fedora"}
   ["dlm-devel"] {os-distribution = "oraclelinux"}
-  ["dlm-git"] {os-distribution = "archlinux"}
   ["libdlm-devel"] {os-distribution = "opensuse"}
 ]
-available: [ os = "linux" ]
+available: [ os = "linux" & os != "alpine" & os != "archlinux"]
```

Comment:
```suggestion
available: [ os = "linux"]
```
Is it possible to install the package by manually compiling the appropriate library?

Reply:
The source code can be found in https://pagure.io/dlm/blob/master/f/libdlm

I guess it's possible to manually compile it, although I haven't tried